### DISCUSSION TITLE
Keine Verbindung mehr zum Hoymiles

### DISCUSSION THREAD

**Discussion by Farquarl** [hoylabs/OpenDTU-OnBattery#975] [Q&A] [answered]
Moin moin,

ich bekomme seit ein paar tagen keine Verbindung mehr zum Hoymiles (HMS-800-2T). Lief bis dahin super. 
DTU ist nur 1 meter entfernt und der gleichrichter speißt auch ein, nur eben ohne DTU kontrolle.

Ich habe bis jetzt versucht:

- Hoymiles ausschalten (neustarten)
- openDTU neustarten
- openDTU neu flashen (webinterface)

Hatte jemand das schon mal von euch? gibt es etwas was ich noch versuchen kann?

**Comment by Farquarl** [comment, marked as answer]
Anscheinend habe ich ein altes Pin_mapping file genommen. (Ein Komma zu viel)
Jetzt läuft es wieder.

**Comment by Gururom** [comment]
Kenn ich, bezweifle das es daran lag, ich experimentiere auch seit drei Tagen, mal garkeine Verbindung , mal nur alle 2-3 Minuten, mal wieder alle 30 sec und irgandwann wieder alle 5sec wie es sein sollte.
Dann bleibts wieder einen ganzen Monat und irgendwann gehts wieder los,
Ich habe noch keine Lösung gefunden, lass mich mittlerweile nicht mehr stören dadurch
(WR HMS 16oo 4T)

**Reply by spcqike** [reply]
Mir ist auch aufgefallen dass meine DTU die Verbindung verliert. Unregelmäßig. Allerdings nur, wenn der DPL aktiv ist. 

Interessanter weise ist das einzige, was ich machen muss, die „DTU“ Einstellungen erneut speichern (also die Frequenz, Leistung etc.) 
Einen Neustart benötige ich nirgends. Weder bei der DTU noch beim wechselrichter. 

Meistens passiert das auch nur einmal irgendwann am Morgen.

**Reply by sentinel1111** [reply]
Kann ich bestätigen. Seit 3 Tage habe ich auch ein HMS1600.
Nachts wenn Batterienutzung Stop-Schwellwert erreicht ist wird der WR gestoppt, startet aber erst wenn Batterienutzung Start-Schwellwert ereicht wird. Das ist meist gegen Mittag erreicht. In der Ereignisanzeige steht nur "Durch Fernbedienung abgeschaltet" und
in Live Ansicht ist der Balken gelb.
Vorher hatte ich ein HMS 800 da ist mir dieses Verhalten nicht aufgefallen, da kamen auch die Meldungen "Durch Fernbedienung abgeschaltet" aber dauerten nur paar Sekunden nicht wie jetzt Stunden.
In der Console steht :

10:22:36.577 > [DPL::loop] ******************* ENTER **********************
10:22:36.916 > [DPL::loop] battery interface enabled, SoC: 63 %, StartTH: 80 %, StopTH: 25 %, SoC age: 0 s, ignore: no
10:22:37.167 > [DPL::getBatteryVoltage] BMS: 50.65 V, MPPT: -1.00 V, inverter: 51.10 V, returning: 50.65V
10:22:37.373 > [DPL::loop] dcVoltage: 50.65 V, loadCorrectedVoltage: 50.65 V, StartTH: 50.50 V, StopTH: 49.00 V
10:22:37.784 > [DPL::loop] StartTH reached: no, StopTH reached: no, SolarPT enabled, use at night: no
10:22:38.193 > [DPL::calcPowerLimit] battery use prevented, solar power (DC): 0 W
10:22:38.395 > [DPL::announceStatus] no energy source available to power the inverter from
10:22:38.743 > [DPL::announceStatus] the system is stable, the last power limit is still valid
10:22:38.953 > Fetch inverter: 1164A00CC0E4
10:22:39.163 > TX RealTimeRunData 865.00 MHz --> 15 A0 0C C0 E4 80 17 09 68 80 0B 00 66 90 E7 AE 00 00 00 00 00 00 00 00 AB D5 21 
10:22:39.418 > Interrupt received
10:22:39.625 > RX 865.00 MHz --> 95 A0 0C C0 E4 80 17 09 68 01 00 01 01 FF 01 FF 00 01 00 00 00 06 00 04 00 00 E8 | -10 dBm
10:22:40.034 > Interrupt received

**Reply by spcqike** [reply]
@sentinel1111 wo ist dein Problem? Das, was du beschreibst, sollte doch genau so passieren, oder? Nachts entlädt er bis zum SoC von 25%, dann muss er aufladen bis SoC 80%. in deinem Log Auszug von 10:22 Uhr hat die Batterie erst einen SoC von 63%, weshalb der WR noch nicht gestartet wird. 

Das nicht Starten, weil der Grenzwert noch nicht erreicht wurde, hat nichts mit keiner Kommunikation zum WR zu tun.

Du scheinst auch keinen Victron MPPT eingebunden zu haben. Sonst könntest du die Solarleistung direkt nutzen und würdest nicht den Umweg über die Batterie gehen.

**Reply by sentinel1111** [reply]
Hallo spcqike,

Doch ich habe ein Victron 150/35 eingebunden.Ich habe nur den Wechselrichter gewechselt weil der HMS800 defekt war.
An den Einstellungen habe ich nichts bewusst geändert.
Mit den HMS800 wurde nicht gewartet bis SOC 80 erreicht wird.

[VE.Direct MPPT 40/39] serial input (317 Bytes):
20:51:03.982 > [VE.Direct MPPT 40/39] 3a 37 44 42 45 44 30 30 45 38 30 44 39 31 0a 3a
20:51:04.282 > [VE.Direct MPPT 40/39] 37 45 43 45 44 30 30 46 46 46 46 37 37 0a 3a 37
20:51:04.589 > [VE.Direct MPPT 40/39] 32 37 32 30 30 30 46 46 46 46 46 46 46 46 30 42
20:51:04.896 > [VE.Direct MPPT 40/39] 0a 3a 41 44 35 45 44 30 30 35 37 31 33 31 46 0a
20:51:05.205 > [VE.Direct MPPT 40/39] 3a 41 42 42 45 44 30 30 33 31 31 38 35 41 0a 3a
20:51:05.510 > [VE.Direct MPPT 40/39] 41 42 44 45 44 30 30 30 30 30 30 41 31 0a 3a 41
20:51:05.824 > [VE.Direct MPPT 40/39] 44 37 45 44 30 30 30 31 30 30 38 36 0a 3a 41 42
20:51:06.131 > [VE.Direct MPPT 40/39] 43 45 44 30 30 45 31 30 32 30 30 30 30 42 46 0a
20:51:06.433 > [VE.Direct MPPT 40/39] 0d 0a 50 49 44 09 30 78 41 30 35 38 0d 0a 46 57
20:51:06.738 > [VE.Direct MPPT 40/39] 09 31 36 34 0d 0a 53 45 52 23 09 48 51 32 32 33
20:51:07.047 > [VE.Direct MPPT 40/39] 36 5a 5a 37 48 45 0d 0a 56 09 34 39 35 30 30 0d
20:51:07.352 > [VE.Direct MPPT 40/39] 0a 49 09 31 30 30 0d 0a 56 50 56 09 36 35 31 34
20:51:07.659 > [VE.Direct MPPT 40/39] 30 0d 0a 50 50 56 09 37 0d 0a 43 53 09 33 0d 0a
20:51:07.966 > [VE.Direct MPPT 40/39] 4d 50 50 54 09 32 0d 0a 4f 52 09 30 78 30 30 30
20:51:08.273 > [VE.Direct MPPT 40/39] 30 30 30 30 30 0d 0a 45 52 52 09 30 0d 0a 4c 4f
20:51:08.569 > [VE.Direct MPPT 40/39] 41 44 09 4f 4e 0d 0a 48 31 39 09 31 31 38 32 36
20:51:08.887 > [VE.Direct MPPT 40/39] 36 0d 0a 48 32 30 09 32 38 35 0d 0a 48 32 31 09
20:51:09.197 > [VE.Direct MPPT 40/39] 31 38 34 36 0d 0a 48 32 32 09 32 32 31 0d 0a 48
20:51:09.508 > [VE.Direct MPPT 40/39] 32 33 09 31 30 36 31 0d 0a 48 53 44 53 09 32 38
20:51:09.810 > [VE.Direct MPPT 40/39] 35 0d 0a 43 68 65 63 6b 73 75 6d 09 79
20:51:10.116 > [VE.Direct MPPT 40/39] Text Data 'PID' = '0XA058'
20:51:10.898 > [VE.Direct MPPT 40/39] Text Data 'FW' = '164'
20:51:10.902 > [VE.Direct MPPT 40/39] Text Data 'SER' = 'HQ2236ZZ7HE'
20:51:11.035 > [VE.Direct MPPT 40/39] Text Data 'V' = '49500'
20:51:11.344 > [VE.Direct MPPT 40/39] Text Data 'I' = '100'
20:51:11.650 > [VE.Direct MPPT 40/39] Text Data 'VPV' = '65140'
20:51:11.958 > [VE.Direct MPPT 40/39] Text Data 'PPV' = '7'
20:51:12.265 > [VE.Direct MPPT 40/39] Text Data 'CS' = '3'
20:51:12.572 > [VE.Direct MPPT 40/39] Text Data 'MPPT' = '2'
20:51:12.881 > [VE.Direct MPPT 40/39] Text Data 'OR' = '0X00000000'
20:51:13.191 > [VE.Direct MPPT 40/39] Text Data 'ERR' = '0'
20:51:13.493 > [VE.Direct MPPT 40/39] Text Data 'LOAD' = 'ON'
20:51:13.802 > [VE.Direct MPPT 40/39] Text Data 'H19' = '118266'
20:51:14.115 > [VE.Direct MPPT 40/39] Text Data 'H20' = '285'
20:51:14.415 > [VE.Direct MPPT 40/39] Text Data 'H21' = '1846'
20:51:14.723 > [VE.Direct MPPT 40/39] Text Data 'H22' = '221'
20:51:15.027 > [VE.Direct MPPT 40/39] Text Data 'H23' = '1061'
20:51:15.340 > [VE.Direct MPPT 40/39] Text Data 'HSDS' = '285'
20:51:15.643 > [VE.Direct MPPT 40/39] Sending Hex Command: :7DBED0086, Free FIFO-Buffer: 118
20:51:26.482 > [VE.Direct MPPT 40/39] Unhandled Hex Async Response, addr: 0xEDD7 (???), value: 0x00000001, flags: 0x00
20:51:26.776 > [VE.Direct MPPT 40/39] Unhandled Hex Async Response, addr: 0xEDBC (???), value: 0x000002E3, flags: 0x00
20:51:26.987 > [VE.Direct MPPT 40/39] Unhandled Hex Async Response, addr: 0xEDD5 (Charger Voltage), value: 0x00001355, flags: 0x00
20:51:27.392 > [VE.Direct MPPT 40/39] Unhandled Hex Async Response, addr: 0xEDBB (Panel Voltage), value: 0x00001787, flags: 0x00
20:51:27.803 > [VE.Direct MPPT 40/39] Unhandled Hex Async Response, addr: 0xEDBD (???), value: 0x00000001, flags: 0x00
20:51:28.210 > [VE.Direct MPPT 40/39] serial input (332 Bytes):
20:51:28.435 > [VE.Direct MPPT 40/39] 3a 37 44 42 45 44 30 30 45 38 30 44 39 31 0a 3a
20:51:28.722 > [VE.Direct MPPT 40/39] 37 45 43 45 44 30 30 46 46 46 46 37 37 0a 3a 37
20:51:29.028 > [VE.Direct MPPT 40/39] 32 37 32 30 30 30 46 46 46 46 46 46 46 46 30 42
20:51:29.233 > [VE.Direct MPPT 40/39] 0a 3a 41 44 37 45 44 30 30 30 32 30 30 38 35 0a
20:51:29.646 > [VE.Direct MPPT 40/39] 3a 41 44 37 45 44 30 30 30 31 30 30 38 36 0a 3a
20:51:30.051 > [VE.Direct MPPT 40/39] 41 42 43 45 44 30 30 45 33 30 32 30 30 30 30 42
20:51:30.257 > [VE.Direct MPPT 40/39] 44 0a 3a 41 44 35 45 44 30 30 35 35 31 33 32 31
20:51:30.668 > [VE.Direct MPPT 40/39] 0a 3a 41 42 42 45 44 30 30 38 37 31 37 30 35 0a
20:51:31.080 > [VE.Direct MPPT 40/39] 3a 41 42 44 45 44 30 30 30 31 30 30 41 30 0a 0d
20:51:31.286 > [VE.Direct MPPT 40/39] 0a 50 49 44 09 30 78 41 30 35 38 0d 0a 46 57 09
20:51:31.492 > [VE.Direct MPPT 40/39] 31 36 34 0d 0a 53 45 52 23 09 48 51 32 32 33 36
20:51:31.899 > [VE.Direct MPPT 40/39] 5a 5a 37 48 45 0d 0a 56 09 34 39 34 39 30 0d 0a
20:51:32.305 > [VE.Direct MPPT 40/39] 49 09 31 30 30 0d 0a 56 50 56 09 36 30 32 33 30
20:51:32.715 > [VE.Direct MPPT 40/39] 0d 0a 50 50 56 09 37 0d 0a 43 53 09 33 0d 0a 4d
20:51:33.124 > [VE.Direct MPPT 40/39] 50 50 54 09 32 0d 0a 4f 52 09 30 78 30 30 30 30
20:51:33.405 > [VE.Direct MPPT 40/39] 30 30 30 30 0d 0a 45 52 52 09 30 0d 0a 4c 4f 41
20:51:33.607 > [VE.Direct MPPT 40/39] 44 09 4f 4e 0d 0a 48 31 39 09 31 31 38 32 36 36
20:51:33.945 > [VE.Direct MPPT 40/39] 0d 0a 48 32 30 09 32 38 35 0d 0a 48 32 31 09 31
20:51:34.356 > [VE.Direct MPPT 40/39] 38 34 36 0d 0a 48 32 32 09 32 32 31 0d 0a 48 32
20:51:34.864 > [VE.Direct MPPT 40/39] 33 09 31 30 36 31 0d 0a 48 53 44 53 09 32 38 35
20:51:35.173 > [VE.Direct MPPT 40/39] 0d 0a 43 68 65 63 6b 73 75 6d 09 76
20:51:35.586 > [VE.Direct MPPT 40/39] Text Data 'PID' = '0XA058'
20:51:35.992 > [VE.Direct MPPT 40/39] Text Data 'FW' = '164'
20:51:36.264 > [VE.Direct MPPT 40/39] Text Data 'SER' = 'HQ2236ZZ7HE'
20:51:36.477 > [VE.Direct MPPT 40/39] Text Data 'V' = '49490'
20:51:36.681 > [VE.Direct MPPT 40/39] Text Data 'I' = '100'
20:51:37.040 > [VE.Direct MPPT 40/39] Text Data 'VPV' = '60230'
20:51:37.243 > [VE.Direct MPPT 40/39] Text Data 'PPV' = '7'
20:51:37.629 > [VE.Direct MPPT 40/39] Text Data 'CS' = '3'
20:51:37.834 > [VE.Direct MPPT 40/39] Text Data 'MPPT' = '2'
20:51:38.245 > [VE.Direct MPPT 40/39] Text Data 'OR' = '0X00000000'
20:51:38.451 > [VE.Direct MPPT 40/39] Text Data 'ERR' = '0'
20:51:38.859 > [VE.Direct MPPT 40/39] Text Data 'LOAD' = 'ON'
20:51:39.268 > [VE.Direct MPPT 40/39] Text Data 'H19' = '118266'
20:51:39.691 > [VE.Direct MPPT 40/39] Text Data 'H20' = '285'
20:51:39.895 > [VE.Direct MPPT 40/39] Text Data 'H21' = '1846'

**Reply by spcqike** [reply]
> 10:22:37.167 > [DPL::getBatteryVoltage] BMS: 50.65 V, MPPT: -1.00 V, inverter: 51.10 V, returning: 50.65V

in dieser Zeile aus deinem log weiter oben steht, dass die MPPT Spannung -1V beträgt. 

> 10:22:38.193 > [DPL::calcPowerLimit] battery use prevented, solar power (DC): 0 W

hier steht dass die Batterie nicht entladen wird (Start SoC nicht erreicht, das steht weiter oben) und dass es 0W Solar power, also von einem MPPT gibt. 

Daher startet der DPL den wechselrichter nicht. 

Hast du an der Verbindung zum MPPT was geändert? Wird der gut und durchgängig erkannt? 

Solange der DPL dort keine Werte hat, wird das mit Solar passthrough nichts. 

Wenn das beim HM-800 noch ging, ist es auch verständlich, dass der WR schon vorm erreichen des start SoC lief.

**Reply by sentinel1111** [reply]
An der Verbindung zum MPPT habe ich nichts geändert.Ich Loge die Werte mit Grafana,da sind keine Aussetzer zu sehen.
Das einzige was ich noch gemacht habe, war das ich eine neuere OpenDtu Version installiert habe.
Ich werde mir morgen alles noch mal genau ansehen, bestimmt habe ich was verstellt.

**Reply by sentinel1111** [reply]
Gestern habe ich mir alles nochmal angesehen.
Als ich die neue OpenDTU Version kompiliert habe, habe ich auch in der Pinmapp Datei zwei Victrons eingetragen rx und rx1.
Habe rx1 rausgenommen und schon funktioniert alles wie es soll, also keine Verbindungsprobleme, sondern eine falsche Konfiguration meinerseits.

Die Meldungen "Durch Fernbedienung abgeschaltet" gibt es weiterhin. aber nur kurz ,so wie davor beim HMS800 auch.

**Reply by SW-Niko** [reply]
Es kann sein das bei mehreren MPPTs immer die kleinere Spannung an den DPL weitergegeben wird. 
Also einen 2ten MPPT konfiguriert, der aber nicht vorhanden ist ergibt -1V. Muss ich mir im Code mal genauer anschauen.

**Reply by spcqike** [reply]
@SW-Niko kann es da auch Probleme mit der gesamt Solar Leistung geben? Die Spannung würde ja die pass through Leistung nicht überschreiben, oder?

**Reply by SW-Niko** [reply]
Hallo @spcqike ,
wenn ich den Code richtig lese dann ist die Gesamtleistung auch betroffen. Ich kann das in den nächsten Tagen mal an meiner Anlage testen.

**Reply by SW-Niko** [reply]
Hallo @spcqike und @sentinel1111 ,
ich konnte das Problem auf meiner Anlage reproduzieren. Habe einen PR #1118 losgetreten um das zu beheben.

**Reply by spcqike** [reply]
Kann die Gesamtleistung denn betroffen sein? So wie ich deinen PR lese, ist das Problem ja, dass min() der verbundenen laderegler zurück gegeben wurde. 

Die Leistung sollte doch aber ein einzelner Wert (aus dem Netzwerk? Wird da das gesamte übertragen?) Oder die Summe der Einzelleistungen sein. Oder? 😄

**Reply by SW-Niko** [reply]
Ja auch die Gesamtleistung ist betroffen.

 if (!config.PowerLimiter.SolarPassThroughEnabled
            || isBelowStopThreshold()
            || !VictronMppt.isDataValid()) {
        return 0;
    }
    auto solarPower = VictronMppt.getPowerOutputWatts();

Die Funktion VictronMppt.isDataValid() erzwingt einen Abbruch und VictronMppt.getPowerOutputWatts() wird nicht mehr aufgerufen.

VictronMppt.getPowerOutputWatts() liefert die Leistung aller MPPTs die mit VE.Direkt verbunden sind.

**Comment by Gururom** [comment]
So, habe heute mal wieder einen Verbindungsverlust, teilweise für ne halbe Stunde, teilweise länger, teilweise kürzer.
Anders als an den anderen Tagen, heute ist starkes Regenwetter, stört eventuell aus welchen Gründen auch immer die Verbindung.
Mein Logfile sieht so aus:
RX Period End
10:39:37.123 > All missing
10:39:37.220 > Nothing received, resend whole request
10:39:37.308 > TX AlarmData 866.00 MHz --> 15 93 52 47 42 80 15 71 64 80 11 00 66 AF 3D CF 00 00 00 00 00 00 00 00 7E 80 05 
10:39:37.396 > Websocket: [/livedata][3] disconnect
10:39:37.787 > RX Period End
10:39:37.939 > All missing
10:39:38.040 > Nothing received, resend whole request
10:39:38.236 > TX AlarmData 866.00 MHz --> 15 93 52 47 42 80 15 71 64 80 11 00 66 AF 3D CF 00 00 00 00 00 00 00 00 7E 80 05 
10:39:38.599 > RX Period End
10:39:38.860 > All missing
10:39:38.960 > Nothing received, resend whole request
10:39:39.014 > TX AlarmData 866.00 MHz --> 15 93 52 47 42 80 15 71 64 80 11 00 66 AF 3D CF 00 00 00 00 00 00 00 00 7E 80 05 
10:39:39.144 > PowerMeterClass: TotalPower: 355.94
10:39:39.204 > [DPL::announceStatus] inverter is offline (polling enabled? radio okay?)
10:39:39.359 > RX Period End
10:39:39.475 > All missing
10:39:39.575 > Nothing received, resend count exeeded
10:39:39.665 > TX SystemConfigPara 866.00 MHz --> 15 93 52 47 42 80 15 71 64 80 05 00 66 AF 3D CF 00 00 00 00 00 00 00 00 6A 94 11 
10:39:39.788 > RX Period End
10:39:39.887 > All missing
10:39:39.972 > Nothing received, resend whole request
10:39:40.039 > TX SystemConfigPara 866.00 MHz --> 15 93 52 47 42 80 15 71 64 80 05 00 66 AF 3D CF 00 00 00 00 00 00 00 00 6A 94 11 
10:39:40.107 > RX Period End
10:39:40.188 > All missing
10:39:40.291 > Nothing received, resend whole request
10:39:40.480 > TX SystemConfigPara 866.00 MHz --> 15 93 52 47 42 80 15 71 64 80 05 00 66 AF 3D CF 00 00 00 00 00 00 00 00 6A 94 11 
10:39:40.543 > RX Period End
10:39:40.700 > All missing
10:39:40.804 > Nothing received, resend whole request
10:39:40.894 > TX SystemConfigPara 866.00 MHz --> 15 93 52 47 42 80 15 71 64 80 05 00 66 AF 3D CF 00 00 00 00 00 00 00 00 6A 94 11 
10:39:40.960 > RX Period End
10:39:41.111 > All missing
10:39:41.212 > Nothing received, resend whole request
10:39:41.301 > TX SystemConfigPara 866.00 MHz --> 15 93 52 47 42 80 15 71 64 80 05 00 66 AF 3D CF 00 00 00 00 00 00 00 00 6A 94 11 
10:39:41.420 > RX Period End
10:39:41.548 > All missing
10:39:41.723 > Nothing received, resend count exeeded
10:39:41.828 > Fetch inverter: 116493524742
10:39:41.931 > Request SystemConfigPara
10:39:42.121 > TX ChannelChangeCommand 868.00 MHz --> 56 93 52 47 42 80 15 71 64 02 15 21 18 14 28 
10:39:42.195 > RX Period End
10:39:42.259 > All missing
10:39:42.343 > Nothing received, resend whole request
10:39:42.428 > TX ChannelChangeCommand 868.00 MHz --> 56 93 52 47 42 80 15 71 64 02 15 21 18 14 28 
10:39:42.519 > RX Period End
10:39:42.851 > All missing
10:39:42.955 > Nothing received, resend whole request
10:39:43.044 > TX ChannelChangeCommand 868.00 MHz --> 56 93 52 47 42 80 15 71 64 02 15 21 18 14 28 
10:39:43.111 > RX Period End
10:39:43.263 > All missing
10:39:43.364 > Nothing received, resend whole request
10:39:43.451 > TX ChannelChangeCommand 868.00 MHz --> 56 93 52 47 42 80 15 71 64 02 15 21 18 14 28 
10:39:43.571 > RX Period End
10:39:43.671 > All missing
10:39:43.876 > Nothing received, resend whole request
10:39:43.965 > TX ChannelChangeCommand 868.00 MHz --> 56 93 52 47 42 80 15 71 64 02 15 21 18 14 28 
10:39:44.080 > RX Period End
10:39:44.184 > All missing
10:39:44.287 > Nothing received, resend count exeeded
10:39:44.376 > TX RealTimeRunData 866.00 MHz --> 15 93 52 47 42 80 15 71 64 80 0B 00 66 AF 3D D8 00 00 00 00 00 00 00 00 54 70 D2 
10:39:44.452 > RX Period End
10:39:44.695 > All missing
10:39:44.799 > Nothing received, resend whole request
10:39:44.885 > TX RealTimeRunData 866.00 MHz --> 15 93 52 47 42 80 15 71 64 80 0B 00 66 AF 3D D8 00 00 00 00 00 00 00 00 54 70 D2 
10:39:44.959 > RX Period End
10:39:45.103 > All missing
10:39:45.307 > Nothing received, resend whole request
10:39:45.398 > TX RealTimeRunData 866.00 MHz --> 15 93 52 47 42 80 15 71 64 80 0B 00 66 AF 3D D8 00 00 00 00 00 00 00 00 54 70 D2 
10:39:45.463 > RX Period End
10:39:45.531 > All missing
10:39:45.705 > Nothing received, resend whole request
10:39:45.809 > TX RealTimeRunData 866.00 MHz --> 15 93 52 47 42 80 15 71 64 80 0B 00 66 AF 3D D8 00 00 00 00 00 00 00 00 54 70 D2 
10:39:45.879 > RX Period End
10:39:46.027 > All missing
10:39:46.131 > Nothing received, resend whole request


Meiner Meinung nach nur zu Erkennen das keine Verbindung zustande kommt.
Habe an der Frequenze bischen was geändert, ohne Erfolg.
Stelle jetzt wieder auf 865 zurück und lass es laufen.

Der Wechselrichter produziert jedoch weiter wie bisher, heute durch das schlechte Wetter zwar wenig, aber er funktioniert...

**Reply by Gururom** [reply]
so, im Moment tingelt er wieder zwischen 10 und 15 Sekunden, bei eingestellten 5 Sec.
Hier noch ein Logfileausschnitt.
Woher kommt eigentlich dieses "Frame kaputt"?

Fetch inverter: 116493524742
13:25:54.434 > TX RealTimeRunData 865.00 MHz --> 15 93 52 47 42 80 15 71 64 80 0B 00 66 AF 64 CD 00 00 00 00 00 00 00 00 AA 51 41 
13:25:54.564 > Websocket: [/livedata][4] disconnect
13:25:54.997 > RX Period End
13:25:55.094 > All missing
13:25:55.147 > Nothing received, resend whole request
13:25:55.206 > TX RealTimeRunData 865.00 MHz --> 15 93 52 47 42 80 15 71 64 80 0B 00 66 AF 64 CD 00 00 00 00 00 00 00 00 AA 51 41 
13:25:55.564 > RX Period End
13:25:55.624 > All missing
13:25:55.676 > Nothing received, resend whole request
13:25:55.725 > TX RealTimeRunData 865.00 MHz --> 15 93 52 47 42 80 15 71 64 80 0B 00 66 AF 64 CD 00 00 00 00 00 00 00 00 AA 51 41 
13:25:55.775 > Interrupt received
13:25:55.833 > Frame kaputt
13:25:55.913 > [DPL::announceStatus] the system is stable, the last power limit is still valid
13:25:55.960 > Interrupt received
13:25:56.014 > Frame kaputt
13:25:56.124 > RX Period End
13:25:56.177 > All missing
13:25:56.227 > Nothing received, resend whole request
13:25:56.280 > TX RealTimeRunData 865.00 MHz --> 15 93 52 47 42 80 15 71 64 80 0B 00 66 AF 64 CD 00 00 00 00 00 00 00 00 AA 51 41 
13:25:56.335 > Interrupt received
13:25:56.392 > Frame kaputt
13:25:56.441 > Interrupt received
13:25:56.494 > Frame kaputt
13:25:56.545 > Interrupt received
13:25:56.595 > Frame kaputt
13:25:56.644 > Interrupt received
13:25:56.695 > RX 865.00 MHz --> 95 93 52 47 42 80 15 71 64 85 00 03 F3 50 F4 | -89 dBm
13:25:56.744 > RX Period End
13:25:56.796 > Middle missing
13:25:56.844 > Request retransmit: 1
13:25:56.894 > TX RequestFrame 865.00 MHz --> 15 93 52 47 42 80 15 71 64 81 D0 
13:25:56.944 > RX Period End
13:25:56.992 > Middle missing
13:25:57.048 > Request retransmit: 1
13:25:57.110 > TX RequestFrame 865.00 MHz --> 15 93 52 47 42 80 15 71 64 81 D0 
13:25:57.158 > RX Period End
13:25:57.208 > Middle missing
13:25:57.263 > Request retransmit: 1
13:25:57.311 > TX RequestFrame 865.00 MHz --> 15 93 52 47 42 80 15 71 64 81 D0 
13:25:57.358 > RX Period End
13:25:57.416 > Middle missing
13:25:57.477 > Request retransmit: 1
13:25:57.531 > TX RequestFrame 865.00 MHz --> 15 93 52 47 42 80 15 71 64 81 D0 
13:25:57.587 > Interrupt received
13:25:57.648 > Frame kaputt
13:25:57.700 > RX Period End
13:25:57.861 > Middle missing
13:25:57.909 > Request retransmit: 1
13:25:58.064 > TX RequestFrame 865.00 MHz --> 15 93 52 47 42 80 15 71 64 81 D0 
13:25:58.166 > RX Period End
13:25:58.268 > Middle missing
13:25:58.317 > Retransmit timeout

**Reply by Martin0475** [reply]
Ich kenne das Problem mit dem "All Missing" auch. 
Meiner Meinung nach passiert das wenn die DTU zu viele Kommandos in zu kurzer Zeit losschickt.
Z.B. wenn der DPL nicht mit regeln hinterher kommt weil das Ceran Kochfeld alle 15 Sekunden an und aus geht.
Zudem habe ich einen Hichi Lesekopf der nur sehr schach empfangen wird. ( Router zu weit weg) . Wenn der auf die Abfrage per JSON nicht rechtzeitig antwortet fängt die OpenDTUoB auch an zu spinnen.

Meisst ist dann das Webinterface extrem träge und es kommt öfter zu einer "time calibration"

Oft fängt sich das System wieder wenn man "Einstellungen", "DTU" aufruft, nichts verändert und nur auf Speichern klickt.

Das AC Kabel des Wechselrichters stört anscheinend auch die Funk Verbindung , vor allem wenn ein sehr niedrieges Limit eingestellt ist. Seit ich es weiter von der Antenne des WR entfernt habe,  gibt es so gut wie keine "Missing Frames" mehr.

WR ist HMS-2000-4T

Gruß Martin

**Reply by Gururom** [reply]
Kann ich bei mir eigentlich auch ausschließen.

Das mit Einstellungen aufrufen und speichern funktioniert bei mir auch nicht.

Jedoch kann es durchaus sein, das es durch Störungen im Funk dazu kommt.
Wodurch ausgelöst, kann ich jedoch nicht verifizieren.

**Reply by Martin0475** [reply]
Könntest du mal beobachten bei welchem Limit des Wechselrichters die Störungen auftreten?

**Reply by Gururom** [reply]
wie meinst du Limit?
Ich habe den DPL eingestellt, so dass er bis voll auf regelt wenn notwendig.
Minimal Limit habe ich 40 Watt eingestellt, (so schaltet der Wechselrichter nicht ab,)
Hysterese glaub ich so  auf -10 und +25 watt. **Korrektur 30W steht bei mir als Hysterese und -50 als Netzbezug**
Das läuft ja auch zu 98 % sauber durch, nur alle paar Monate halt die Verbindungsaussetzer

**Reply by spcqike** [reply]
> Könntest du mal beobachten bei welchem Limit des Wechselrichters die Störungen auftreten?

mir sind in der Vergangenheit Verbindungsprobleme zu meinem HMS-2000 aufgefallen, wenn er >800W liefert. Welches Limit dabei eingestellt war, kann ich nicht sagen. 

Hier zB 
![image](https://github.com/user-attachments/assets/4b7e0ca9-9acb-47fd-aff2-1f56ea6bb30e)


Aktuell läuft s aber seit mehreren Tagen stabil. Raus gefunden hab ich es bisher noch nicht, woran es liegen könnte.

**Reply by Martin0475** [reply]
Mit Limit meine ich den Wert den OpenDTU dem Wechselrichter vorgibt.

Ich glaube bei bestimmten Watt-Werten die der Wechselrichter liefert, erzeugt er selbst ungewollte Störsignale.

Evtl könnte man ja DPL mal ausschalten und verschiedene Werte testen.
In der Weboberfläche läuft ja die Zeit seit der letzten Aktualisierung. 
Bei mir z.B. alle 7 Sekunden. Wenn der dann bis 12, 15 oder 20, etc zählt, weiss ich die Verbindung ist schlecht.


Seit ich das AC Kabel des Wechselrichters anders verlegt habe, meine ich die Störungen sind fast weg. Kann mich aber auch täuschen, aktuell läuft es recht stabil

**Reply by Gururom** [reply]
Also damit habe ich definitiv keine Probleme, bei allen Watt Werten steht die Verbindung einwandfrei, auch die Nähe zur AC Leitung sehe ich nicht als Problem, da dies bei mir auch , wie gesagt, zu 98,9 % sauber läuft.
Es kommt halt nur alle 30-40 Tage schätze ich, zu den vermeintlichen Abbrüchen der Verbindung.
Kann das weder an deinen Kriterien festmachen, genau so wie nicht an Wetter, temperatur, Leistung oder irgendwas.
Einfach nicht erklärbar.
Zumal das dann spätestens nach einem Tag wieder erledigt ist, 
Nein auch nicht durch Abschaltung und Neustart verifizierbar, auch nicht über einen längeren Zeitraum.
Ich belass es jetzt einfach dabei

**Reply by Martin0475** [reply]
Nachdem ich gestern noch 2x Marstek Speicher installiert habe und dazu den Wechselrichter verrutscht habe, bzw die DC Kabel anders verlegt habe,  ist der Hoymiles heute  3x hängengeblieben. 

Ich konnte es sogar provozieren:
Wenn ich den WR auf 45% stelle empfängt die DTU nichts mehr.

Wenn ich meine Hand um die Antenne des WR lege (wie ein Reflektor) wechselt die Anzeige in Open DTU innerhalb weniger Sekunden wieder von rot auf blau und die Daten kommen in regelmäßigen Abständen rein.

Bei Leistungswerten um die 200 Watt kommt das Signal auch so sauber rein..

Aber bei höheren Werten kann die OpenDTU das Signal nicht mehr sauber empfangen. 

WR und DTU sind ca. 15 Meter entfernt.  Allerdings sind 2 Aussenwände dazwischen (vom Haus und der Garage).  OpenDTU im Haus,  WR in der Garage.


In der Konsole wird mir ein Pegel von -56dbm angezeigt.  Das sollte eigentlich ein guter Wert sein.

**Reply by Gururom** [reply]
WR auf 45%  lässt eher einen Wackler im Gerät vermuten, oder aber die Sendeempfangsleistung ist so Grenzwertig, das leichtes verrücken reicht.
Wieviel db ist denn eigentlich sinnvoll, bzw in welche Richtung sollte der db-Wert gehen??
ca 15 Meter und eine dicke Wand aus Stein und Beton sinds bei mir auch, was aber ebend über Wochen null Probleme macht und dann mal für einen Tag die Aussetzer.
Ich habe auch keinerlei Probleme mit hohen Leistungen, dauerhaft.
Manches mal liegt über Stunden 1,4kw an, geht problemlos
